### PR TITLE
feat(options): add support for different manifest input than bundle.dest

### PR DESCRIPTION
**Pull request by msc117**

### CODE REVIEW

**Comment by phamann** [comment]
Hey @msc117 :wave: thanks for the PR. Can you describe a scenario when someone might need to use a different key in the manifest other than the original bundle destination?

**Comment by msc117** [comment]
In my use case, I'm using `path.resolve` for `bundle.dest`. This causes the manifest to include the full file path as the input, i.e. `/Users/msc117/Projects/...` which is not ideal for my project.

File: README.md [hunk 1]
```diff
@@ -85,6 +85,15 @@ Example manifest:
 }
 ```
 
+### input
```
Comment (phamann) [inline README.md]: As `input` is quite an ambiguous property name, especially in the context of Rollup. Can you change this to something more specific, such as `manifestInput`, `manifestKey` or `manifestProperty`?

File: README.md [hunk 2]
```diff
@@ -85,6 +85,15 @@ Example manifest:
...
+
+Type: `string`
+Default: `bundle.dest`
+Required: `false`
+
+The name of your filename intro used in creating manifest.
```
Comment (phamann) [inline README.md]: Change to something along the lines of: "The filename used as the input key in the generated manifest map."

File: README.md (hunk 1 above)
Comment (msc117) [inline README.md]: Going to switch it to `manifestKey`

Comment (phamann) [inline README.md]: 👍

File: README.md (hunk 2 above)
Comment (msc117) [inline README.md]: Updated

**Comment by phamann** [comment]
Thanks for your help!

**Comment by phamann** [comment]
I've now published this as `v.1.2.0`

**Comment by msc117** [comment]
Thanks!

**Pull request merged**